css/menu.css

```diff
@@ -252,3 +252,7 @@ body.disableScroll {
 [data-fl-exit-app] .fa-angle-right.linked-icon {
   display: none;
 }
+
+.hidden {
```

Comment:
Why is this needed?

![image](https://user-images.githubusercontent.com/290733/109194115-02733500-7791-11eb-84dd-5b010a5afb7e.png)

Bootstrap already has this exact same style.

Reply:
@tonytlwu removed unneeded class